crates/proof-of-sql/Cargo.toml

```diff
@@ -75,7 +75,8 @@ default = ["arrow", "perf"]
 arrow = ["dep:arrow", "std"]
 blitzar = ["dep:blitzar", "dep:merlin", "std"]
 test = ["dep:rand", "std"]
-perf = ["blitzar", "rayon", "ark-ec/parallel", "ark-poly/parallel", "ark-ff/asm"]
+perf = ["blitzar", "cpu-perf"]
+cpu-perf = ["rayon", "ark-ec/parallel", "ark-poly/parallel", "ark-ff/asm"]
```

Comment:
This isn't directly related to this example. I just added it here as a convenience going forward.

Reply:
Looks good thank you